lib/oban/plugins/cron.ex

```diff
@@ -1,20 +1,19 @@
 defmodule Oban.Plugins.Cron do
   @moduledoc """
-  Periodically enqueue jobs through CRON based scheduling.
-
-  This plugin registers workers a cron-like schedule and enqueues jobs automatically. Periodic
-  jobs are declared as a list of `{cron, worker}` or `{cron, worker, options}` tuples.
+  Periodically enqueue jobs through [cron](https://en.wikipedia.org/wiki/Cron)-based scheduling.
```

Comment:
Moved some stuff around here, I like the flow now, thoughts?

Reply:
Agreed, the new flow is an improvement!